jupyter_server/extension/application.py

```diff
@@ -252,8 +259,13 @@ def launch_instance(cls, argv=None, **kwargs):
         # Initialize the server
         serverapp = cls.initialize_server()
 
+        # Handle arguments.
+        if argv is not None:
+            args = sys.argv[1:]  # slice out extension config.
+        else:
+            args = argv
```

Comment:
The else case basically says `args = None`. Is that the intention, or should it be `args = []`?
I suggest to write down the actual value, rather than using `argv` that can only be None at that point.

Reply:
You're right. This should be `args = []`.